OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -46,10 +49,8 @@ public class ArmamentInfo : UpgradableTraitInfo, ITraitInfo, Requires<AttackBase
 
 		[Desc("Muzzle flash sequence to render")]
 		public readonly string MuzzleSequence = null;
-
```

Comment:
Usually I'd agree, but we're talking about 5 new lines in a class he already changes, 2 of which are being removed by him. Making another PR for those would probably cause the code to be left with no new lines (pretty much how it was left the last time someone changed these files).
Plus it seems redundant.